packages/override_api_endpoint/lib/src/override_api_endpoint.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:uni_links/uni_links.dart';
+
+Future<Uri> overrideApiEndpoint(
+  String deeplinkOverrideSegment,
+  String deeplinkQueryParameter,
+  Uri defaultEndpoint,
+) async {
+  const apiEndpointKey = 'apiAddress';
```

Comment:
I'd make this also configurable in the class mentioned above, with a default value

Reply:
I see no benefit in making this configurable

Reply:
YAGNI, let's resolve this one